src/components/task/index.cjsx

```diff
@@ -54,24 +42,24 @@ module.exports = React.createClass
   render: ->
     {id} = @props
     model = TaskStore.get(id)
-    steps = TaskStore.getSteps(id)
-    stepConfig = steps[@state.currentStep]
+    stepConfig = @state.steps[@state.currentStep]
```

Comment:
why store all the steps in the `state`? (Does the component need to be updated when new props are provided?)